editor-core/src/main/webapp/app/core/editorCore/model/DiagramScene.ts

```diff
@@ -32,15 +32,15 @@ class DiagramScene extends joint.dia.Paper {
             }),
             validateConnection: function (cellViewS, magnetS, cellViewT, magnetT, end, linkView) {
                 return (!(magnetT && magnetT.getAttribute('type') === 'output')
-                     && !(cellViewT && cellViewT.model.get('type') === 'link'));
```

Comment:
this file contains no changes except for formatting, so I think they should be reverted